src/dev/i18n/extractors/html.js

```diff
@@ -139,14 +142,19 @@ function* getFilterMessages(htmlContent) {
       throw createFailError(`Empty "id" value in angular filter expression is not allowed.`);
     }
 
-    const { message, context } = parseFilterObjectExpression(filterObjectExpression) || {};
+    const { message, context, valuesKeys } =
+      parseFilterObjectExpression(filterObjectExpression, messageId) || {};
 
     if (!message) {
       throw createFailError(
         `Empty defaultMessage in angular filter expression is not allowed ("${messageId}").`
       );
     }
 
+    if (valuesKeys) {
```

Comment:
__question:__ it feels we should turn this (and others) check the other way around - now we do value check only if values object is provided, but what if label includes some parameters, but values object isn't provided at all? Parsing _every_ message may be too expensive though, but we can probably do simple `defaultMessage.includes('{')` check or something like that before calling `parser.parse` (with a proper clarifying comment).

So basically, if basic default message checks passed we _always_ call `checkValues(messageId: string, message: string, values?: Record<string, any>)` that will check if `message.includes('{') === false` first etc. etc. And we throw if:

* Label includes some values, but `values` object isn't provided
* Label includes some values and they don't match exactly to what is included into `values` object
* Label doesn't include any values, but `values` object is provided

 What do you think?